.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 conf/local.js
 *.log
 .DS_Store
+.vscode
```

Comment:
.gitignore allows environment specific exclusions and that is the purpose of it. 

Anyone that works with Visual Studio Code will commit its IDE specific files if this line is not in .gitignore

If you aware for other way to exclude these files from the commit please share...